content/blog/getting-started-with-pulumi-audit-logs/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Getting Started with Pulumi Audit Logs"
+date: "2020-02-10"
+meta_desc: "Get started with audit logs"
+meta_image: "pulumi-webhooks.png"
+authors: ["sean-holung"]
+tags: []
```

Comment:
Can I just add a tag here called "Audit Logs" or something similar, or is there like some predefined set of values I should be referencing?

Reply:
Maybe "enterprise"?